ansible/roles/secor/defaults/main.yml

```diff
@@ -24,4 +24,208 @@ partition_prefix_mapping:
 sunbird_account_name: 
 sunbird_account_key: 
 sunbird_secor_container_name: 
-secor_zip_path: ../../target
\ No newline at end of file
+secor_zip_path: ../../target
+
+secor_service_name:
+  
+  sl_observation_druid:
+    consumer_group: "{{ env }}.sl.observation.druid.backup"
+    base_path: "observation_druid"
+    timestamp_key: "syncts"
+    fallback_timestamp_key: "@timestamp"
+    topic: "{{ env }}.sl.observation.druid"
+    kafka_broker_host: "{{groups['processing-cluster-kafka']|join(':9092,')}}:9092"
+    zookeeper_quorum: "{{groups['processing-cluster-zookeepers']|join(':2181,')}}:2181"
+    max_file_size: "100000000"
+    max_file_age: "14400"
+    ostrich_port: "9931"
+    partition_prefix_enabled: "false"
+    partition_prefix_key: ""
+    partition_prefix_mapping: "{}"
+    output_file_pattern: "{partition}-{currentTimestamp}.json"
+    service_description: "Sl Observation Druid backup"
+    service_monitor_schedule_hr: "23"
+    service_monitor_schedule_min: "50"
+    message_channel_identifier: ""
+    message_parser: "com.pinterest.secor.parser.PatternDateMessageParser"
+  sl_observation_evidence_druid:
```

Comment:
@Kaali09 Please check this once.

Reply:
@rjshrjndrn This is fine but we already running secor job in kubernetes. you may need to work on updating the secor configuration to deploy into kubernetes. 

I am not sure about the decision taken in design discussion. Please check with SanthoshG once.

Reply:
@Kaali09 which file @sivakarthikeyanoptit have to update the values ?